content/topics/javascript-debugging-in-vscode/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+content_type: topic
+ready: true
+tags:
+  - vscode
+  - javascript
+title: Javascript debugging in VSCode
+---
+
+### Introduction to Debugging in Visual Studio Code
```

Comment:
Heading levels matter.

This code will render into the following:

```
<h1>Javascript debugging in VSCode</h1>
<h3>Introduction to Debugging in Visual Studio Code</h3>
```

That h3 should be an h2.  

Please always pay attention to heading levels.

This is partially because it's important to use html elements as they are meant to be used because it makes things more accessible. And we are at some point going to move away from Hugo and it would be good not to have to reformat everything because heading levels are random.